tx-source/site_core.php

```diff
@@ -241,11 +248,14 @@
 $Definition['Are you sure you want to leave this conversation?'] = 'Are you sure you want to leave this conversation?';
 $Definition['Are you sure you want to leave this group?'] = 'Are you sure you want to leave this group?';
 $Definition['Are you sure you wish to continue?'] = 'Are you sure you wish to continue?';
+$Definition["Article"] = "Article";
```

Comment:
These source strings are being moved into knowledge base.

https://github.com/vanilla/vanilla/pull/8756
https://github.com/vanilla/knowledge/pull/967

We should move them into the knowledge resource.